xml/installation-installation-designate-designate_migrate_zone_pool_to_worker_producer.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xml:id="DNS-MIGRATE"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Migrate Zone/Pool to Worker/Producer after Upgrade</title>
+ <para>
+  To migrate the zone/pool scheme to the worker/producer scheme after upgrade
+  , follow these steps.
+ </para>
+  <procedure>
+   <step>
+    <para>
+     Log in to the &clm;.
+    </para>
+   </step>
+   <step>
+    <para>
+     Edit the
+     <filename>~/openstack/my_cloud/definitions/data/control_plane.yml</filename> 
+     file to replace designate-pool-manager to designate-worker and designate-zone-manager to designate-producer
```

Comment:
I would tweak the initial wording to `Edit the file <filename> to replace....`

also wrap the strings being replaced and the replacements in <literal> ... so  `<literal>designate-pool-manager</literal>`  ... I'll have to check with the doc team if they prefer some other highlighting, but we definitely want to highlight the specific strings